docs/application/vstools/install.md

```diff
@@ -1,170 +1,102 @@
-  - JDK 9 will be supported soon.
-
-### Emulator Requirements
-
-The Tizen emulator for Visual Studio has the same requirements as the emulator in the Tizen Studio. To check the detailed hardware and software requirements for the Tizen emulator, see [Emulator Requirements](../tizen-studio/setup/prerequisites.md#emulator).
-
-- Intel&reg; Hardware Acceleration Execution Manager (Intel&reg; HAXM) speeds up the Tizen emulation on Intel-VT-enabled systems. The Intel&reg; HAXM installation is started automatically as part of the Visual Studio Tools for Tizen installation. For more information, see [Hardware Accelerated Execution Manager](../tizen-studio/setup/hardware-accelerated-execution-manager.md).
+	![Visual Studio Tools for Tizen](media/prerequisite-vs.png)
 
-- Make sure **Hyper-V** is disabled (in Windows&reg; 10 or higher):
-  1. In the **Start** menu, select **Programs and Features**.
+> **Note**
+>
+> Visual Studio Tools for Tizen works with all Visual Studio variations, including Community. Installing or re-installing Visual Studio with .NET desktop development and .NET Core cross-platform development toolsets is recommended.
 
-   ![Programs and Features](media/cs_prerequisite01-250x401.png)
+# Visual Studio Tools for Tizen 
 
-  2. Select **Turn Windows features on or off**.
-  3. Disable **Hyper-V** and click **OK**.
-
-   ![Disable Hyper-V](media/cs_prerequisite-disable-hiperv.png)
-
-  4. Reboot the computer.
+To use the Tizen SDK tools in Visual Studio, you must install:
 
+1.	[Visual Studio Tools for Tizen Extension](#installing-visual-studio-tools-for-tizen-extension )
+2.	[Tizen Baseline SDK](#installing-tizen-baseline-sdk)
 
 <a name="install"></a>
-## Visual Studio Tools for Tizen Installation
-
-To use the Tizen SDK tools, you must install the Visual Studio Tools for Tizen extension and the baseline SDK.
-
-> **Note**  
-> If the Tizen Studio is already installed in your computer, simply [set the Tizen baseline SDK path](#set-tizen-baseline-sdk) to the existing SDK instead of reinstalling it.
-
-
-### Installing the Extension
-
-The Visual Studio Tools for Tizen extension is registered in the Visual Studio Marketplace. You can install extensions from the Visual Studio Marketplace in the Visual Studio IDE.
-
-1. In the Visual Studio IDE menu, go to **Tools &gt; Extensions and Updates**.
-2. In the Visual Studio Marketplace, search for **Tizen**.
-
-   ![Marketplace](media/howtoinstall-extensionwindows.png)
-
-3. Click **Download** and close the Visual Studio IDE.
-
-   The installation starts.
-
-Alternatively, you can download the Visual Studio Tools for Tizen extension from the [Visual Studio Marketplace Web site](https://marketplace.visualstudio.com/items?itemName=tizen.VisualStudioToolsforTizen).
+## Install Visual Studio Tools for Tizen Extension 
 
```

Comment:
Information Required: Could you provide some more details to give an overview. Why do we need to install, what is the purpose.